src/docs/Components/Form/PhoneNumberInput/code.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Code'
+order: 2
+---
+
+# Import
+
+Import **the settings**, the **text-input** and the **phone-number** scss files.
+
+```scss
+@import 'settings-tools/_all-settings';
+@import 'components/_c.text-input';
+@import 'components/_c.phone-number';
+```
+
+# Usage
+A phone number input is composed of 2 main zones: 
+
+- *A dropdown* with the class : `mc-phone-number__dropdown`
+- *A number input * with the class : `mc-phone-number__input`
+
+```html
+<div class="mc-phone-number">
+  <div class="mc-phone-number__dropdown">
+    <button
+      aria-haspopup="listbox"
+      aria-labelledby="dropdown_country"
+      id="dropdown_country"
+      class="mc-phone-number__button"
+    >
+      <svg class="mc-phone-number__flag">
+        <use xlink:href="#flag" />
+      </svg>
+      <span>+33</span>
+    </button>
+    <ul
+    id="phone_number_list"
+      tabindex="-1"
+      role="listbox"
+      aria-labelledby="phone_number_list"
+      class="mc-phone-number__list mc-phone-number__list--hidden"
+    >
+      <li id="FR" role="option" class="mc-phone-number__item">
+        <svg class="mc-phone-number__flag">
+          <use xlink:href="#flag" />
+        </svg>
+        <span class="mc-phone-number__country">France, </span>
+        <span>+33</span>
+      </li>
+    </ul>
+  </div>
+  <input
+    type="text"
+    class="mc-phone-number__input mc-text-input mc-text-input--m mc-field__input"
+    id="smallField"
+    placeholder="00 00 00 00 00"
+    name="example-small"
+  />
+</div>
+```
+
+<Preview path="basic" />
+
+# Detail of areas
+## Dropdown
+The dropdown contains: 
+ - The dropdown trigger (`mc-phone_number_button`)
+ - The list of the countries available (`mc-phone_number_list`)
+
+Each element of list contains: 
+ - The country flag ('mc-phone_number_flag)
+ - The country name ('mc-phone_number_country)
+ - The country indicator
+
+# Variation
+## With label
+
+<Preview path="label" />
+
+# Behaviour
+The javascript to manage the dropdown is based on the listbox from [W3C](https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/listbox/listbox-collapsible.html)
+
+To have a fully fonctionnal phone number input, make sure to add the following script to your code.
+
+[listbox.js](https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/listbox/js/listbox.js)
+[utils.js](https://www.w3.org/TR/wai-aria-practices-1.1/examples/js/utils.js)
```

Comment:
Can you put all this part in an `<Highlight theme="tips">` or `<Highlight theme="warning">`